database/postgres/schema.sql

```diff
@@ -0,0 +1,14 @@
+CREATE DATABASE updatessdc;
+
+CREATE TABLE PROJECT(id INTEGER PRIMARY KEY NOT NULL,createdAt timestamp);
+
```

Comment:
new lines could make this easier to read

Reply:
Sure